src/appleseedmaya/exporters/shadingnodeexporter.cpp

```diff
@@ -469,18 +469,9 @@ void ShadingNodeExporter::exportValue(
     }
     else if (paramInfo.paramType == "float")
     {
-        if (paramInfo.mayaAttributeType == "angle")
```

Comment:
We cannot remove this code. 
IIRC, if you don't use MAngle, you get the result value in radians.

I opened a PR adding string units = "degrees" to shader params and 
removing the last uses of maya_attribute_type = "angle" we have.
After merging it I think we can use the units metadata to bring back this case.